python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -346,6 +357,40 @@ async def configure(self, config):
                 DeprecationWarning,
             )
 
+        self.time_on_target = config.time_on_target
+        self.time_on_target = (
+            self.compute_time_on_target()
+            if config.time_on_target is None
+            else config.time_on_target
+        )
+
+    def compute_time_on_target(self):
+        """Determine the amount of time spent on target so the rotator position
+        can be optimized. Assumed slew time is 3 minutes, and that the read,
+        calculation, and offset totals 20s.
+
+        Returns
+        -------
+        self.time_on_target: float
+            Estimate of time on target in seconds.
+        """
+
+        time_on_target = 180 + self.max_acq_iter * (self.acq_exposure_time + 20)
+        if self.do_take_sequence:
+            nexp = len(self.visit_configs)
+            sci_expTimeTotal = 0.0
+            for (filt, expTime, grating) in self.visit_configs:
```

Comment:
`camelCase` in `expTime`.. use `exp_time`

Reply:
used `exptime` as per the rest of the code.
Again, as noted below, this was changed throughout so you'll see many more changes.